main.tf

```diff
@@ -120,6 +120,21 @@ module "dosomething-dev" {
   papertrail_destination_fastly_dev = "${var.papertrail_destination_fastly_dev}"
```

Comment:
is "_dev" indicates _dev environment? if so, it shouldn't be hardcoded in template and be taken as a var.

Reply:
Good point, renamed to `papertrail_destination_fastly` in both places in #55.